Integrations/MongoDB/MongoDB.py

```diff
@@ -0,0 +1,312 @@
+    client: Client, collection: str, entry: str, **kwargs
+) -> Tuple[str, dict, list]:
+    # test if query is a valid json
+    try:
+        entry_json = json.loads(entry)
+        if not isinstance(entry_json, list):
+            entry_json = [entry_json]
+        entries = convert_id_to_object_id(entry_json)
+        results = client.insert_entry(collection, entries)  # type: ignore[arg-type]
+        if not results.acknowledged:
+            return_error("MongoDB: Error occurred when trying to enter insert entries.")
+            sys.exit(1)
+        else:
+            object_ids = convert_object_id_to_str(results.inserted_ids)
+            human_readable = tableToMarkdown(
+                f"MongoDB: Successfully entered {len(object_ids)} entry to the '{collection}' collection.",
+                object_ids,
+                headers=["_id"],
+            )
+            outputs = {CONTEXT_KEY: [{"_id": _id} for _id in object_ids]}
+            return human_readable, outputs, object_ids
+    except JSONDecodeError:
+        return_error("MongoDB: The `entry` parameter is not a valid json.")
+        sys.exit(1)
+
+
+def update_entry_command(
+    client: Client,
+    collection: str,
+    filter: str,
+    update: str,
+    update_one=False,
+    **kwargs,
+) -> Tuple[str, None]:
+    try:
+        json_filter = json.loads(filter)
+        json_filter = convert_id_to_object_id(json_filter)
+    except JSONDecodeError:
+        return_error("MongoDB: The `filter` parameter is not a valid json.")
+        sys.exit(1)
+    try:
+        json_update = json.loads(update)
+    except JSONDecodeError:
+        return_error("MongoDB: The `update` parameter is not a valid json.")
+        sys.exit(1)
+    response = client.update_entry(
+        collection, json_filter, json_update, argToBoolean(update_one)
+    )
+    if not response.acknowledged:
+        return_error("MongoDB: Error occurred when trying to enter update entries.")
+    return (
+        f"### MongoDB: Total of {response.modified_count} entries has been modified.",
+        None,
+    )
+
+
+def delete_entry_command(
+    client: Client, collection, filter, delete_one, **kwargs
+) -> Tuple[str, None]:
+    try:
+        json_filter = json.loads(filter)
+        json_filter = convert_id_to_object_id(json_filter)
+    except JSONDecodeError:
+        return_error("MongoDB: The `filter` parameter is not a valid json.")
+        sys.exit(1)
+    results = client.delete_entry(collection, json_filter, argToBoolean(delete_one))
+    if not results.acknowledged:
+        return_error("MongoDB: Error occurred when trying to enter delete entries.")
+    return f"### MongoDB: Delete {results.deleted_count} entries.", None
+
+
+def create_collection_command(client: Client, collection: str, **kwargs) -> Tuple[str, None]:
+    if client.is_collection_in_db(collection):
+        return f"### MongoDB: Collection '{collection}' is already exists", None
+    _ = client.create_collection(collection)
```

Comment:
why are we initalizing here `_`? just dont return anything

Reply:
it's returning collection. I'm by assigning this to nothing the output of the function is not printed to the console.

Reply:
so if you dont return anything, what does it do?